src/styles/blocks/main.scss

```diff
@@ -0,0 +1,91 @@
+.main {
```

Comment:
So is it main or cards block?
Remove styles
Also you do not need this font-family here, set it to body/html tag